src/logic/operate.js

```diff
@@ -0,0 +1,19 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const operations = {
+    '+': 'add',
+    '-': 'minus',
+    '÷': 'div',
+    '×': 'times',
+  };
```

Comment:
We should also take care of `%` in this file. 
* [ ] `operate` switches `operation` and uses `plus`, `minus`, `times`, `div` and `modifier` (turning the number into a percentage based number, `e.g. 95 % == .95`) functions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))